### ISSUE TITLE
Can't import it as a module

### ISSUE THREAD

**Issue by scarfacedeb** [dixso/custombox#238]
It seems to be missing module definition in compiled and minified version.

Therefore I can't import it using `import Custombox from 'custombox'`.

Requiring custombox.ts with ts-loader brought a lot of other issues.

**Comment by scarfacedeb** [comment]
Surprisingly, version 3.0.2 had the module definitions.

**Comment by dixso** [comment]
Hi @scarfacedeb,
What do you want do exactly?

Reagards.

**Comment by zeleniy** [comment]
In console:
```
npm install custombox --save
```
In code:
```
import Custombox from 'custombox';

new Custombox.modal({
  content: {
    effect: 'fadein',
    target: '#popup'
  }
}).open();
```
And then finally in browser:
```
Uncaught TypeError: __WEBPACK_IMPORTED_MODULE_1_custombox___default.a.modal is not a constructor
```

**Comment by hqa11** [comment]
@zeleniy  the same to you,Have you solved this problem?Please help me..Thanks

**Comment by zeleniy** [comment]
@hqa11 no, i used https://github.com/frontend-collective/react-image-lightbox instead of that tool

**Comment by HeshamMeneisi** [comment]
If someone has this issue with webpack. Use "yarn add https://github.com/HeshamMeneisi/custombox" until someone accepts the pull request.

**Comment by hunterInt** [comment]
Any update here?

**Comment by gp0** [comment]
Any update here?

**Comment by hunterInt** [comment]
@gp0  this is what I had to do:
```
require('./assets/vendor/custombox/dist/custombox.legacy.min.js');
```

PS commander keen is boss
